lib/sensu/api/process.rb

```diff
@@ -334,7 +334,17 @@ def resolve_event(event_json)
           unless clients.empty?
             clients.each_with_index do |client_name, index|
               settings.redis.get("client:#{client_name}") do |client_json|
-                response << MultiJson.load(client_json)
+                begin
+                  response << MultiJson.load(client_json)
+                rescue MultiJson::ParseError
```

Comment:
That's true. It seems unlikely that we would have a partial or invalid json object in there. Originally I had no idea what was happening so I just caught everything.